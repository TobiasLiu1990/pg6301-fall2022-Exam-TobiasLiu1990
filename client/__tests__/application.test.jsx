import React from "react";
import ReactDOM from "react-dom";
import { Application } from "../application";

describe("application component", () => {
  it("should render", () => {
    const element = document.createElement("div");
    ReactDOM.render(<Application />, element);

    expect(element.innerHTML).toMatchSnapshot();
  });
});
