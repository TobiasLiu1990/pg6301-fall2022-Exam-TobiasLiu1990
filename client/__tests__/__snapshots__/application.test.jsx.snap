// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application component should render 1`] = `"<div>Page loading...</div>"`;
